dshr/dshr_mod.F90

```diff
@@ -1068,8 +1068,7 @@ subroutine dshr_restart_write(rpfile, case_name, model_name, inst_suffix, ymd, t
 
     call shr_cal_datetod2string(date_str, ymd, tod)
     write(rest_file_model ,"(7a)") trim(case_name),'.', trim(model_name),trim(inst_suffix),'.r.', trim(date_str),'.nc'
-    ! temporarily turn off timestamp, remove this code and comment in alpha05c
-    lrpfile = rpfile(:len_trim(rpfile)-17)
+
     ! write restart info to rpointer file
     if (my_task == main_task) then
        open(newunit=nu, file=trim(lrpfile), form='formatted')
```

Comment:
Hi @jedwards4b 

Sorry - me again, confused again.

I think at line 1074, you meant to change `lrpfile` back to `rpfile` ?